enhancements/windows-containers/windows-node-egress-proxy.md

```diff
@@ -0,0 +1,341 @@
+---
+title: windows-node-cluster-wide-egress-proxy
+authors:
+  - "@saifshaikh48"
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
+  - "@openshift/openshift-team-windows-containers"
+  - TBD -- sdn?
```

Comment:
Meant to say this before. If we do need other reviews it would be the @openshift/openshift-team-network-edge